site/pages/admin/user-management.page.tsx

```diff
@@ -0,0 +1,115 @@
+        return (
+            userGroupName.charAt(0).toUpperCase() +
+            (userGroupName.toLowerCase().endsWith("s") ? userGroupName.slice(1, -1) : userGroupName.slice(1))
+        );
+    };
+
+    const getRows = () => {
+        const rows: { header?: string | ReactNode; cells: string[] | ReactNode[] }[] = [];
+        getDataInPages(currentPage, userList).forEach((user, index) => {
+            rows.push({
+                cells: [
+                    `${getOrgDisplay(user.group)}`,
+                    user.email,
+                    user.userStatus === "CONFIRMED" ? "Active" : "Pending invite",
+                    createLink("user-action", index, user.userStatus === "CONFIRMED" ? false : true),
+                ],
+            });
+        });
+        return rows;
+    };
+
+    const createLink = (key: string, index?: number, sendInvite?: boolean) => {
+        return sendInvite ? (
+            <>
+                <Link key={`${key}${index ? `-${index}` : ""}`} className="govuk-link" href="/">
+                    Resend invite
+                </Link>
+                <br />
+                <Link key={`${key}${index ? `-remove-${index}` : "-remove"}`} className="govuk-link" href="/">
+                    Remove
+                </Link>
+            </>
+        ) : (
+            <Link key={`${key}${index ? `-${index}` : ""}`} className="govuk-link" href="/">
+                Remove
+            </Link>
+        );
+    };
+
+    return (
+        <BaseLayout title={title} description={description}>
+            <>
+                <div className="govuk-form-group">
+                    <h1 className="govuk-heading-xl">User Management</h1>
+                </div>
+                <Table
+                    caption={{ text: "Users", size: "m" }}
+                    columns={["Account type", "User email", "Status", "Action"]}
+                    rows={getRows()}
+                ></Table>
+                <Link role="button" href={"/add-user"} className="govuk-button--secondary govuk-button mt-5">
+                    Add new user
+                </Link>
+
+                <PageNumbers
+                    numberOfPages={numberOfUserPages}
+                    currentPage={currentPage}
+                    setCurrentPage={setCurrentPage}
+                />
+            </>
+        </BaseLayout>
+    );
+};
+
+export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props: UserManagementPageProps }> => {
+    if (!ctx.req) {
+        throw new Error("No context request");
+    }
+
+    const sessionWithOrg = await getSessionWithOrgDetail(ctx.req);
+    const userRecords = await listUsersWithGroups();
+
+    let userList: UserManagementSchema = [];
+    const reducedList: UserManagementSchema = [];
+
+    if (userRecords) userList = userManagementSchema.parse(userRecords);
+
+    userList = userList.reduce((reducedUserList, user) => {
```

Comment:
Or to be honest thinking about it we could probably get rid of the map entirely and just the return the org as a separate prop since it'll be the same for everyone:

```ts
const userRecords = await listUsersWithGroups();

if (!userRecords) {
    return {
        props: {
            userList: [],
        },
    };
}

const userList = userManagementSchema
    .parse(userRecords)
    .filter((user) => user.organisation === sessionWithOrg.orgId);

return {
    props: { userList, organisation: sessionWithOrg.orgName },
};
```

Reply:
Or as mentioned, just get rid of org here